### PR TITLE
Create unit tests for order util methods. Closes #9

### LINKED ISSUE #9: Create Unit Tests for Order Util Methods

### PR DESCRIPTION
**Pull request by LauraCallahan**
Initial unit tests for util methods created. Still need to refactor code to skip records with errors and will update testing once code is refactored.

### CODE REVIEW

File: src/test/java/Ahead/JSONDataProcessing/util/OrderUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                    () -> assertEquals(255.0, result.getTotal_order_value(), 0.001));
+        }
+    }
+
+    @DisplayName("Test Sum Digits - ")
+    @Nested
+    class TestSetSumDigits {
+        @DisplayName("With Zero")
+        @Test
+        void testSumDigitsWithZero() {
```
Comment (joaomelloahead) [inline src/test/java/Ahead/JSONDataProcessing/util/OrderUtilTest.java:46]: if sumDigits gets a null value still working or return "0"?

File: src/test/java/Ahead/JSONDataProcessing/OrderTests.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+public interface OrderTests {
+
+    OrderData orderData = new OrderData();
+    ArrayList<Order> orderArrayListOne = new ArrayList<>();
+    ArrayList<Order> orderArrayListMultiple = new ArrayList<>();
+    @BeforeAll
+    default void beforeAll() {
+        orderArrayListOne.clear();
+        orderArrayListMultiple.clear();
+
```
Comment (joaomelloahead) [inline src/test/java/Ahead/JSONDataProcessing/OrderTests.java:20]: I just can see test for the happy path, I'm not seeing test for the bad path as a null-value, fieldname missing or wrong...

File: src/test/java/Ahead/JSONDataProcessing/OrderTests.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                .product("Product 1")
+                .quantity(3)
+                .unit_price(10.0)
+                .build();
+        Order order2 = Order.builder()
+                .product("Product 2")
+                .quantity(5)
+                .unit_price(25.0)
+                .build();
+        Order order3 = Order.builder()
```
Comment (joaomelloahead) [inline src/test/java/Ahead/JSONDataProcessing/OrderTests.java:31]: I think you should create new objects with different wrong data and add them to the list after you do the happy path.

_```
Order order4 = Order.builder()
                .product("Product 3")
                .quantity(5) 
                .**price**(50.0)
                .build();
```_

**Review by joaomelloahead — CHANGES_REQUESTED** [review]

**Comment by LauraCallahan** [comment]
@joaomelloahead I've added testing for null values. Please review when you have a moment.

**Review by joaomelloahead — APPROVED** [review]

**Pull request merged**